tv_apps/dashboard/manifest.webapp

```diff
@@ -19,5 +19,10 @@
     "336": "/style/icons/dashboard_336.png",
     "672": "/style/icons/dashboard_672.png"
   },
-  "orientation": "landscape"
+  "orientation": "landscape",
+  "permissions": {
+    "browser": {},
+    "embed-widgets": {},
+    "webapps-manage": {}
```

Comment:
The code to enumerate widgets can be found at:
https://github.com/huchengtw-moz/gaia/blob/58921c5dfebfc1998dd2ecd4a7308439782ac69e/dev_apps/homescreen-stingray/js/applications.js

You may trim them and bring widget handling to `shared/smart-screen/applications.js`.